src/components/achievements/Stat.astro

```diff
@@ -44,10 +44,16 @@ const query = `${subPath}?q=${isType}+${userFilter}`
               class="pl-2"
             >
               <span
-                class="block w-[var(--bar-w)] min-w-[2px] bg-blue-purple-gradient h-2 rounded-full"
+                class="block w-[var(--bar-w)] min-w-[2px] bg-blue-purple-gradient h-2 rounded-full group relative"
                 style={`--bar-w: ${percentage}%;`}
               >
                 <span class="sr-only">{repoCount} {type}</span>
+                <span
+                  aria-hidden="true"
+                  class="invisible absolute z-50 -mt-8 rounded bg-gray-100 p-1 text-neutral-700 opacity-0 transition-opacity duration-300 ease-in-out group-hover:visible group-hover:opacity-100"
+                >
+                  {repoCount.toString()}
+                </span>
```

Comment:
I think it might be good to use the existing `sr-only` element here instead of adding an extra one as the current approach impacts accessibility: a screenreader would read for example something like “starlight 22 reviews 22” instead of the current “starlight 22 reviews”.

Tailwind has a [`not-sr-only` class](https://tailwindcss.com/docs/screen-readers#undoing-screen-reader-only-elements) for undoing `sr-only` styles, so you could probably use a similar approach to what you have now, but applying that class when the hover kicks in.